### ISSUE TITLE
gmsh finite element pre and post processor

### ISSUE THREAD

**Issue by alen12345** [conda-forge/staged-recipes#5179]
paraview, a very powerful post-processing tool for FEA is already on conda-forge; I would like to add gmsh, too, especially for parametric mesh generation.

@jakirkham assign this to me please ;)

**Comment by bgruening** [comment]
Done! Paraview-Galaxy user here, would like to see gmsh as well :)

**Comment by alen12345** [comment]
@bgruening we would also need opencascde to build solid geometry... I will try to build it on my machine and then create a recipe for conda-forge!

**Comment by looooo** [comment]
occt is here: https://github.com/conda-forge/occt-feedstock

**Comment by looooo** [comment]
any news on this? I see there are some good packages allready available:
- https://anaconda.org/guyer/gmsh
- https://anaconda.org/guyer/fltk

@guyer any interest in a conda-forge package for gmsh?

**Comment by guyer** [comment]
@looooo Absolutely. I just need find time to make sure these work across all platforms and most importantly (to me) work for my own software.
